code/modules/mob/living/carbon/human/species_types/monthmen.dm

```diff
@@ -0,0 +1,76 @@
+/datum/species/monthmen
+	//an exotic species that arrives once a year to remove the worst species, mothpeople.
+	name = "Monthman"
+	id = "month"
+	//visuals
+	default_color = "FFFFFF"
+	species_traits = list(NO_UNDERWEAR, NOBLOOD, ABSTRACT_HEAD)
+	default_features = list("mcolor" = "FFF", "tail_human" = "None", "ears" = "None", "wings" = "None")
+	skinned_type = /obj/item/paper
+	changesource_flags = NONE //absolutely no way to get the race, it's completely out of theme of ss13 and should stay locked up in april fools
```

Comment:
Travis will fail with this pr until it's merged because it fixes the unit test in the same pr that fails the unit test

Reply:
Pretty sure it uses the unit test of the current PR